kano_profile/tracker.py

```diff
@@ -107,6 +107,7 @@ def generate_tracker_token():
 OS_VERSION = str(read_file_contents('/etc/kanux_version'))
 CPU_ID = str(get_cpu_id())
 TOKEN = load_token()
+LANGUAGE = (os.getenv('LANG') or '').split('.', 1)[0]
```

Comment:
@skarbat I have also somehow encountered `None` values for getting the `LANG` value, hence:

The following can throw an exception if the key in not in the dict:
```
os.environ['LANG']`
```

However, the following doesn't throw an exception, but may return `None` instead:
```
os.getenv('LANG')
```